listenbrainz/config.py.sample

```diff
@@ -89,11 +89,11 @@ PLAYING_NOW_MAX_DURATION = 10 * 60
 #    'level': 'ERROR',
 # }
 
-# LOG_SENTRY = {
-#    'dsn':'',
-#    'environment': 'development',
-#    'level': 'ERROR',
-# }
+LOG_SENTRY = {
+    'dsn':'https://examplePublicKey@o0.ingest.sentry.io/0',
```

Comment:
Is this a real test dsn that people can use? 
I'd prefer that during local development we don't configure a DSN, and only enable it in js if the value is non-null.

Reply:
I copied this DSN from sentry docs but guarding the configuration seems fine to me will do that.